doc/whatsnew.rst

```diff
@@ -4,6 +4,7 @@ What's new
 Next release
 ============
 
+- Introduce a SSP configuration for cooling technologies with share constraints (:pull:`256`).
```

Comment:
```suggestion
- Introduce an SSP configuration for cooling technologies with share constraints (:pull:`256`).
```
Isn't it `an`? I think `a`/`an` are based on sound and if you say `SSP` (the abbreviation), you say `es es pee`, sort of.